ddtrace/contrib/dbapi/__init__.py

```diff
@@ -61,7 +61,7 @@ def __enter__(self):
         # previous versions of the dbapi didn't support context managers. let's
         # reference the func that would be called to ensure that errors
         # messages will be the same.
-        self.__wrapped__.__enter__()
+        self.__wrapped__.__enter__
```

Comment:
I think it's worth a comment why `__enter__` is not being called no?  This makes `__enter__` a no-op